nuitka/nodes/BuiltinAnyNodes.py

```diff
@@ -29,14 +29,12 @@
 class ExpressionBuiltinAny(ExpressionBuiltinSingleArgBase):
     """ Builtin Any Node class.
 
-    Parameter
-    ---------
-    ExpressionBase: 'any' expression
-
-    Returns
-    -------
-    Bool: True if at least one element of an iterable is true
-          else return False
+    Args:
+        ExpressionBase: 'any' expression
+
+    Returns:
```

Comment:
I don't think classes should have a `Returns` ... we do not have a good example for a class, but I would say 

```
Returns:
    Node that represents built-in 'any' call.
```